main-module/src/main/java/com/diffblue/javatest/StringStuff.java

```diff
@@ -0,0 +1,346 @@
+package com.diffblue.javatest;
+
+import java.io.PrintStream;
+import java.util.Locale;
+
+public class StringStuff {
+
+  public static String classGetNameExample(Class c) {
```

Comment:
The method bodies all come directly from the regression tests on CBMC - I'd rather leave them as is.